### PR TITLE
Fix: Compatibility Issue with IP2Location DB using int for IP Range

### PR DESCRIPTION
**Pull request by muneebwaqas416**
This is the branch of Muneeb Waqas SE Intern

### CODE REVIEW

**Comment by muneebwaqas416** [comment]
@UmanShahzad Please review this PR.

**Review by UmanShahzad — CHANGES_REQUESTED** [review]
Still not cleaned up. See your diff

**Comment by muneebwaqas416** [comment]
@UmanShahzad Please check it and do let me know if I have done it right or wrong

File: lib/cmd_import.go [hunk 1]
```diff
@@ -218,6 +219,7 @@ func CmdImport(f CmdImportFlags, args []string, printHelp func()) error {
 		if f.Csv && f.Tsv || f.Csv && f.Json || f.Tsv && f.Json {
 			return errors.New("multiple input file types specified")
 		} else if f.Csv {
+
```
[the remark this answers is not in the document]
Comment (muneebwaqas416) [reply lib/cmd_import.go:222]: This is not done by me .These lines are in master branch

**Comment by muneebwaqas416** [comment]
@UmanShahzad I have incorporated your feedback in latest commit.

File: .gitignore [hunk 2]
```diff
@@ -1,4 +1,4 @@
 .vim/
 tmp/
 dist/*
-!dist/DEBIAN/
+!dist/DEBIAN/
```
Comment (UmanShahzad) [inline .gitignore:4]: Do return the ending newline

File: lib/cmd_import.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (UmanShahzad) [reply lib/cmd_import.go:222]: This is still not undone

**Review by UmanShahzad — CHANGES_REQUESTED** [review]

**Comment by muneebwaqas416** [comment]
@UmanShahzad You can check now with feedback incorporated

**Comment by muneebwaqas416** [comment]
@UmanShahzad Please review it

**Comment by UmanShahzad** [comment]
@muneebwaqas416 can you please squash commits n lemme know when this is ready?

**Comment by UmanShahzad** [comment]
Preferring #26

**Pull request closed without merging**